### DISCUSSION TITLE
Support filtering Catalog by file type

### DISCUSSION THREAD

**Discussion by gjoseph92** [microsoft/PlanetaryComputer#83] [Ideas]
Small ask—I sometimes want to look for all datasets in a given format (COG, zarr, etc.). Searching for [`zarr`](https://planetarycomputer.microsoft.com/catalog?filter=zarr) doesn't give any results. [`COG`](https://planetarycomputer.microsoft.com/catalog?filter=COG) has some, but clearly not all. Probably just tagging datasets consistently would address this.

Since I mostly use PC to make demos / test open-source projects, not do actual analysis, this is maybe not a use case that would be relevant to may others :)

**Comment by TomAugspurger** [comment]
This would indeed be useful. We recently made some changes to our frontend that'll make it possible to add many more keywords (which is what's powering the search). We'll update the collections to include file formats.
